tests/StarkEx.Crypto.SDK.Tests/Signing/StarkExSignerTests.cs

```diff
@@ -105,6 +105,75 @@ public class StarkExSignerTests
+    [InlineData("0x800000000000000000000000000000000000000000000000000000000000003", "0x41bef28265fd750b102f4f2d1e0231de7f4a33900a214f191a63d4fec4e72f4")]
+    [InlineData("0x800000000000000000000000000000000000000000000000000000000000004", "0x24de66eb164797d4b414e81ded0cfa1a592ef0a9363ebbcb440d4d03cb18af1")]
+    [InlineData("0x800000000000000000000000000000000000000000000000000000000000005", "0x5efb18c3bc9b69003746acc85fb6ee0cfbdc6adfb982f089cc63e1e5495daad")]
+    [InlineData("0x800000000000000000000000000000000000000000000000000000000000006", "0x10dc71f00918a8ebfe4085c834d41dd22b251b9f81eef8b9a4fab77e7e1afe9")]
+    [InlineData("0x800000000000000000000000000000000000000000000000000000000000007", "0x4267ebfd379b1c8caae73febc5920b0c95bd6f9f3536f47c5ddad1259c332ff")]
+    [InlineData("0x800000000000000000000000000000000000000000000000000000000000008", "0x6da515118c8e01fd5b2e96b814ee95bad7d60be4d2ba6b47e0d283f579d9671")]
+    [InlineData("0x800000000000000000000000000000000000000000000000000000000000009", "0x7a5b4797f4e56ed1473876bc2693fbe3f2fef7e050717cbae924ff23d426052")]
+    [InlineData("0x2e9c99d8382fa004dcbbee720aef8a97002de0e991f6a8344e6dc636a71b59e", "0x1ff6803ae740e7e596504ac5c6afbea472e53679361e214f12be0155b13e25d")]
+    [InlineData("0x8620458785138df8722214e073a91b8f55076ea78197cf41007692dd27fd90", "0x5967da40b90d7ca1e36dc4024381d7d4b403c6ac1a0ab358b0743984934a805")]
+    [InlineData("0x1b920e7dfb49ba5ada673882af5342e7448d3e9335e0ac37feb6280cd7289ce", "0x78c7ab46333968fbde3201cf512c1eeb5529360259072c459a158dee4449b57")]
+    [InlineData("0x704170dbfd5dc63caef69d2ce6dfc2b2dbb2af6e75851242bbe79fb6e62a118", "0x534bd8d6ebe4bb2f6992e2d7c19ef3146247e10c2849f357e44eddd283b2af6")]
+    [InlineData("0x4b58bf4228f39550eca59b5c96a0cb606036cc9495eef9a546f24f01b1b7829", "0x1097a8c5a46d94596f1c8e70ca66941f2bb11e3c8d4fd58fdc4589f09965be8")]
+    [InlineData("0x2e93226c90fb7a2381a24e940a94b98433e3553dcbf745d3f54d62963c75604", "0x369f0e8c8e984f244290267393a004dba435a4df091767ad5063fece7b1884c")]
+    [InlineData("0x4615f94598cd756ad1a551d7e57fd725916adfd0054eb773ceb482eef87d0b2", "0x1ee5b8d612102a2408cde59ce52a6498d2e38fe8789bb26d400dea310684ec9")]
+    [InlineData("0x6ade54b7debd7ca1d4e8e932f9545f8fa4024d73be1efcc86df86367fc333f8", "0x37de3bf52412b2fb9b0030d232ca9dd921cd8f71fd67975cdc62546826e121")]
+    [InlineData("0x618e7467dd24c2a3449c4df640439c12cdd0f8ea779afcee6e252b2cf494354", "0x71c2b578c432f2d305d3808bb645ecc46dd670cb43d4f4a076f75ccbff74fbc")]
+    [InlineData("0x7eae185e1f41ec76d214d763f0592f194933622a9dd5f3d52d0209f71619c1a", "0x2b0160052e70176e5b0ff2a6eff90896ae07b732fc27219e36e077735abd57e")]
+    [InlineData("0x178047D3869489C055D7EA54C014FFB834A069C9595186ABE04EA4D1223A03F", "0x1895a6a77ae14e7987b9cb51329a5adfb17bd8e7c638f92d6892d76e51cebcf")]
+    public void GetPublicStarkKeyFromPrivateKey_InputsAreValid_PublicStarkKeyMatches(
```

Comment:
Looks like that the StarkWare repo has some unit tests here https://github.com/starkware-libs/starkware-crypto-utils/blob/dev/test/js/key_derivation.spec.js

But we are not testing the same values (at least the ones I checked). I think we should also include them. Wdyt?

Reply:
Which ones are missing? I'm using the same test set they're using [here](https://github.com/starkware-libs/starkware-crypto-utils/blob/dev/test/js/key_derivation.spec.js#L55) which is this json https://github.com/starkware-libs/starkware-crypto-utils/blob/dev/src/config/keys_precomputed.json

Reply:
I missed that file. LGTM, resolving